### ISSUE TITLE
Launch parameter for autopin

### ISSUE THREAD

**Issue by Damglador** [overlayeddev/overlayed#224]
**Is your feature request related to a problem? Please describe.**
It's annoying to always open the Overlayed app, move it and hide the bar.

**Describe the solution you'd like**
KDE Plasma is intrusive in its window control, and Plasma being Plasma it can overwrite every property of any window. That allows user to set and lock Overlayed on top of everything in any place on a screen from the start of the program, but remains the issue of this stupid in-app bar that I have to click "Pin" on (And also move the actual overlay to the left because position for these for some reason doesn't save) and that's annoying. Without this bar, I can just configure window properties in Plasma and throw it in autostartup, so it's always there if or when I need it, or as an alternative throw it in the Discord's .desktop file to start it with Discord.

**Describe alternatives you've considered**
Idk, recompile the app without the bar.

**Additional context**
DE: KDE Plasma
Arch btw

**Comment by Hacksore** [comment]
This seems like it's related 😅

https://github.com/flathub/dev.overlayed.Overlayed/issues/2

**Comment by Damglador** [comment]
> This seems like it's related 😅
> 
> https://github.com/flathub/dev.overlayed.Overlayed/issues/2

Somewhat yes. But the point of the suggestion is to be able to fully automate setup of the overlay after a reboot

**Comment by Hacksore** [comment]
Yeah I want to have settings persist so that both rust and the front end can access them but I need to find some time to lock in and finish #204 

![image](https://github.com/user-attachments/assets/30be53b7-bca5-4acb-8ed0-c28b3bed4691)

**Comment by BLACK4585** [comment]
I saw you closed #204 . Is this good or bad for this issue?

**Comment by Hacksore** [comment]
@BLACK4585 blocked by #244 and i just have not had time to work shipping it yet.
